simc_support/game_data/WowSpec.py

```diff
@@ -370,6 +370,17 @@ def to_dict(self) -> dict:
     full_name="Windwalker",
     simc_name="windwalker",
 )
+MISTWEAVER = WowSpec(
+    id=269,
+    wow_class=WowClass.MONK,
+    translations=Language.EmptyTranslation(),
+    talents_blueprint="1110011",
```

Comment:
I think only the following talent rows are dps relevant for healer monks: `1000000`

Reply:
I was looking at https://www.icy-veins.com/wow/mistweaver-monk-pve-healing-spec-builds-talents

Might have made more sense if someone who actually knows anything about the spec would have done the changes. I should have said so instead of just trying to get it done I guess.

Reply:
I would think that at least 1010011 is relevant for mistweavers, looking at those talent descriptions selected there.

Reply:
I thought only the first talent row actually affects damage done. If that is not the case, sure this talent blueprint is good.

Reply:
The only talents the effect mw dps are 
tier 15 (chi-wave and chi-burst)
tier 40 (diffuse magic if you transfer a debuff back on the target)
tier 45 (Invoke Chi-ji the red crane)
Tier 50 (focus thunder)

All other rows are pointless in terms of dps

Reply:
the talents i would suggest taking are 
2000031
as chi-wave can see play depending on rsk cd and totm stacks
chi-ji is just a dps upgrade
and focus thunder slightly increases rsks per minute which is the biggest damaging spell you cast

Reply:
```suggestion
    talents_blueprint="1000011",
```